src/common/services/SyncService/SyncService.ts

```diff
@@ -0,0 +1,24 @@
+import { EntityManager } from 'typeorm/browser';
+import { ENTITIES } from '../../constants';
+
+import { UtilService } from '../UtilService';
+
+
+class SyncService {
+    private manager: EntityManager;
+
+    constructor(manager: EntityManager) {
+        this.manager = manager;
+    }
+
+    public async log(id: string, type: string, payload: string) {
+        this.manager.save(ENTITIES.SYNC_LOG, {
+            id,
+            type,
+            payload,
+            timestamp: new UtilService().now()
```

Comment:
Not sure if there's a nicer way of getting `UtilService` from the dependency manager?

Reply:
if the above suggestion is made (creating a syncservice instance and injecting into subscribers) then having the `utilService` be a dependency of the `SyncService` can work with just injecting through the constructor.

pretty common in di and allows for easier testing (can inject a util service where `now` returns `0` always - otherwise it's very hard to write a test for)